src/Gui/ViewProvider.h

```diff
@@ -142,6 +142,8 @@ class GuiExport ViewProvider : public App::TransactionalObject
     virtual SoSeparator* getBackRoot() const;
     ///Indicate whether to be added to scene graph or not
     virtual bool canAddToSceneGraph() const {return true;}
+    // Indicate whether to be added to scene graph root or object group root

```

Comment:
The comment isn't very clear. Especially because we expect toe find the `true` value first then the `false` one.
```suggestion
    // Indicate whether to be added object group (true) or only to scene graph (false)
```